test/canvas.spec.js

```diff
@@ -1,7 +1,10 @@
 const http = require('http');
 process.env.NODE_ENV = 'test';
 const app = require('../src/server');
+const idea = require('../src/idea');
 const Browser = require('zombie');
+const chai = require('chai');
```

Comment:
We're using Chai (all of those Node module updates were from adding in Chai). Chai will be used with Mocha & Zombie for Unit tests. Testcafe is on the new branch from @sampritchard  & @Dino982, so they're experimenting with it's installation & setting up the basic canvas feature tests.